.github/workflows/gradle_localstack_postgres_verify.yml

```diff
@@ -7,6 +7,7 @@ on:
         required: false
         default: '21'
       java-options:
+        description: Any JAVA_OPTS that need to be set.  Not normally required when using github actions
```

Comment:
I've copied across all of the latest changes from the `gradle_localstack_postgres_verify.yml` file in the github actions repo